homeassistant/components/gios/__init__.py

```diff
@@ -52,12 +52,11 @@ async def _async_update_data(self):
         """Update data via library."""
         try:
             with timeout(30):
```

Comment:
I'd suggest to store the timeout in a const as well. 
```suggestion
            with timeout(API_TIMEOUT):
```